transports/http/src/main/java/org/mule/transport/http/HttpMessageProcessTemplate.java

```diff
@@ -56,6 +57,7 @@
     private RequestLine requestLine;
     private boolean failureResponseSentToClient;
     private HttpThrottlingHeadersMapBuilder httpThrottlingHeadersMapBuilder;
+    private Map<String, String> extraHeaders = new HashMap<String, String>();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace the type specification in this constructor call with the diamond operator ("<>"). ([squid:S2293](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS2293))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7625&issues=AWiOv-9iotIXnmsU1tFv&open=AWiOv-9iotIXnmsU1tFv)

[ISSUE_KEY]: AWiOv-9iotIXnmsU1tFv